db/model/Image.ts

```diff
@@ -262,3 +250,46 @@ export class Image extends BaseEntity implements ImageMetadata {
         )
     }
 }
+
+export async function getImageByFilename(
+    knex: KnexReadonlyTransaction,
+    filename: string
+): Promise<Image | undefined> {
+    const image = await knex
+        .table("images")
```

Comment:
I think we should use a variable here if we're using one for all our other tables 🙂